nextcord/client.py

```diff
@@ -1909,54 +1965,236 @@ def add_application_command(
         """
         self._connection.add_application_command(command, overwrite=overwrite, use_rollout=use_rollout)
 
-    async def deploy_application_commands(
+    async def sync_all_application_commands(
+            self,
+            data: Optional[Dict[Optional[int], List[dict]]] = None,
+            use_rollout: bool = True,
```

Comment:
Likely make these kwargs only so people dont do the wrong arg thinking its something else?